js/app.js

```diff
@@ -0,0 +1,71 @@
+// Selectors for counters
+const TOTAL = 'total-zombies';
+const DEAD = 'dead-zombies';
+const SMALL = 'small-zombies';
+const MAD = 'mad-zombies';
+const STRONG = 'strong-zombies';
+
+let counters = {
+    [TOTAL]: 0,
+    [DEAD]: 0,
+    [SMALL]: 0,
+    [MAD]: 0,
+    [STRONG]: 0,
+};
+
+let calculateZombies = () => {
+    let totalCounter = 0;
+
+    zombiesData.map((zombieHP) => {
```

Comment:
map returns a new array
so it doesn't make sense to use "map" in this case
it's just a simple "for"

Reply:
fixed